{"archive_info": {"hash": "sha256=327cbda3da756e2de031a3107b81ab7b3770a602c4d16ca618298c526f4bec1e"}, "url": "https://files.pythonhosted.org/packages/e8/14/9c6a7e5f12294ccd6975a45e02899ed25468cd7c2c86f3d9725f387f9f5f/exceptiongroup-1.1.0-py3-none-any.whl"}